scripts/sendpayment.py

```diff
@@ -65,6 +66,7 @@ def main():
     #without schedule file option, use the arguments to create a schedule
     #of a single transaction
     sweeping = False
+    payjoinurl = None
```

Comment:
Now that BIP has been assigned, maybe change this to `bip79url`, as we will have two different payjoin protocols implemented in JM?

Reply:
Yeah but I think you mean bip78.

Reply:
Yes, I meant bip78.

Reply:
Done in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/536/commits/41540ab53e4ae86ffdd02688e689e56df3356229